frontend/src/cards/RateTrendsChartCard.tsx

```diff
@@ -101,8 +121,14 @@ export function RateTrendsChartCard(props: RateTrendsChartCardProps) {
             metricConfigRates.metricId
           ).withData;
 
+        const demographicGroupsLabelled = isCawpCongress
+          ? demographicGroups
+              .map((race) => getWomenRaceLabel(race))
+              .filter((womenRace) => womenRace !== "Women of Unknown Race")
```

Comment:
Interesting... should we? I was just treating them the same as the way we do 100k which is to leave them off of everything except the unknowns map and the unknowns bubbles. but we could plot "Women of Unknown Race" as another line on the trend chart since we are really measuring a pct_share. 

There aren't any unknowns in US Congress, so I had added this filter to keep the "Women of Unknown Race" option from populating in the time chart legend filter section.

Reply:
Are "women of unknown race" different than standard unknowns?

Reply:
No, they are the same. The condition of "identifying as a woman and being a member of us Congress" and this is the pct_share where no race was listed